stream-chat-android-e2e-test/api/stream-chat-android-e2e-test.api

```diff
@@ -6,6 +6,8 @@ public final class io/getstream/chat/android/compose/uiautomator/ActionsKt {
 	public static final fun goToForeground (Landroidx/test/uiautomator/UiDevice;)V
 	public static final fun longPress (Landroidx/test/uiautomator/UiObject2;I)V
 	public static synthetic fun longPress$default (Landroidx/test/uiautomator/UiObject2;IILjava/lang/Object;)V
+	public static final fun retryOnStaleObjectException (Landroidx/test/uiautomator/UiDevice;ILkotlin/jvm/functions/Function0;)Ljava/lang/Object;
+	public static synthetic fun retryOnStaleObjectException$default (Landroidx/test/uiautomator/UiDevice;ILkotlin/jvm/functions/Function0;ILjava/lang/Object;)Ljava/lang/Object;
```

Comment:
To be honest, I don't why we have API generation for the e2e-test module 🤔

Reply:
I think it is there because it is is configured globally on the project build.gradle, but it shouildn't be needed for this module as far as it is only used for testing purpose.
This module can be added on the ignore section

Reply:
FYI: https://github.com/GetStream/stream-chat-android/pull/5648